python/tank/platform/engine.py

```diff
@@ -1256,30 +1256,34 @@ def _apply_external_styleshet(self, bundle, widget):
 
     def _define_qt_base(self):
         """
-        This will be called at initialisation time and will allow 
+        This will be called at initialisation time and will allow
         a user to control various aspects of how QT is being used
         by Tank. The method should return a dictionary with a number
-        of specific keys, outlined below. 
-        
+        of specific keys, outlined below.
+
         * qt_core - the QtCore module to use
         * qt_gui - the QtGui module to use
+        # wrapper - the Qt wrapper root module, e.g. PySide
         * dialog_base - base class for to use for Tank's dialog factory
-        
+
         :returns: dict
         """
-        # default to None
         base = {"qt_core": None, "qt_gui": None, "dialog_base": None}
         try:
-            from PySide import QtCore, QtGui
-            base["qt_core"] = QtCore
-            base["qt_gui"] = QtGui
-            base["dialog_base"] = QtGui.QDialog
+            # Local import since Qt might not be available.
+            from .qt.qt_importer import QtImporter
+            importer = QtImporter()
+            base["qt_core"] = importer.QtCore
+            base["qt_gui"] = importer.QtGui
+            base["dialog_base"] = importer.QtGui.QDialog
+            base["wrapper"] = importer.wrapper
```

Comment:
I introduced the `wrapper` because it was useful in engines that override define_qt_base (like tk-shell) that wanted to print out the name of the wrapper module importer as well as the version.